### PR TITLE
fix(Popover): opacity transition was not happening

### PR DESCRIPTION
**Pull request by robertpenner**
This affects all components that use `createSlideStyles` (e.g. via `Popover` or `MenuPopover`): 
`AvatarGroup`, `InfoLabel`, `Popover`, `TeachingPopover`, `Menu`, `MenuButton`, `SplitButton`, `Breadcrumb`, and various charts.

# Current Behavior

There is no fade during the slide of a Popover. ([public Storybook](https://react.fluentui.dev/?path=/docs/components-popover--docs#default))

https://github.com/user-attachments/assets/6f8e66de-6f4c-42b8-8b56-92c6fef4a038

# New Behavior

The popover transition has a fade-in while sliding. ([PR Storybook](https://fluentuipr.z22.web.core.windows.net/pull/33421/public-docsite-v9/storybook/index.html?path=/docs/components-popover--docs#default))

https://github.com/user-attachments/assets/a9de2cd9-f7c2-4648-8c8a-7b270b8655a0

## Related Issue(s)

- Fixes #[33420](https://github.com/microsoft/fluentui/issues/33420)

### CODE REVIEW

File: packages/react-components/react-positioning/src/createSlideStyles.ts [hunk 1]
```diff
@@ -8,12 +8,13 @@ import { DATA_POSITIONING_PLACEMENT } from './constants';
  * @returns Griffel styles to spread to a slot
```
Comment (fabricteam) [inline packages/react-components/react-positioning/src/createSlideStyles.ts:8]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33421&commitId=a88f80462db6d8df89e1cff8f339fbd9021b89cb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|6| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4072| Changed|
|Drawer.overlay drawer full.chromium.png|1139| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-positioning/src/createSlideStyles.ts [hunk 2]
```diff
@@ -8,12 +8,13 @@ import { DATA_POSITIONING_PLACEMENT } from './constants';
...
 export function createSlideStyles(mainAxis: number): GriffelStyle {
+  // With 'accumulate' animation composition, the opacity keyframes are deltas from the default value of 1.
   const fadeIn = {
     from: {
-      opacity: 0,
+      opacity: -1, // default of 1 - 1 = 0 actual
     },
     to: {
-      opacity: 1,
+      opacity: 0, // default of 1 + 0 = 1 actual
```
Comment (layershifter) [inline packages/react-components/react-positioning/src/createSlideStyles.ts:17]: Let's do undo this, this is a very tricky and fragile fix 💥 

As we have two separate motions, let's allow the browser to do its work:

```diff
-   animationComposition: 'accumulate',
+   animationComposition: 'replace, accumulate',
    animationDuration: tokens.durationSlower,

    // Note: at-rules have more specificity in Griffel
    '@media(prefers-reduced-motion)': {
      [`&[${DATA_POSITIONING_PLACEMENT}]`]: {
+       animationComposition: 'replace',
        animationDuration: '1ms',
        animationName: fadeIn,
      },
    },
    // Tested in Firefox 79
    '@supports not (animation-composition: accumulate)': {
      [`&[${DATA_POSITIONING_PLACEMENT}]`]: {
+       animationComposition: 'replace',
        animationName: fadeIn,
      },
    },
  };
```

**Review by layershifter — CHANGES_REQUESTED** [review]
Great find, but let's improve the fix 👍

File: packages/react-components/react-positioning/src/createSlideStyles.ts (hunk 2 above)
Comment (robertpenner) [reply packages/react-components/react-positioning/src/createSlideStyles.ts]: @layershifter Good suggestion to separate the animation composition modes; done.

**Review by layershifter — APPROVED** [review]

**Pull request merged**